service/src/test/resources/ehr/mapper/observation/expected-output-observation-statement-16.xml

```diff
@@ -0,0 +1,20 @@
+<component typeCode="COMP" >
+    <ObservationStatement classCode="OBS" moodCode="EVN">
+        <id root="394559384658936" />
+        <statusCode code="COMPLETE" />
+        <effectiveTime>
+            <center value="20190601113000"/>
+        </effectiveTime>
+        <availabilityTime value="20190601113000"/>
+        
+        <pertinentInformation typeCode="PERT">
+            <sequenceNumber value="+1" />
+            <pertinentAnnotation classCode="OBS" moodCode="EVN">
+                <text>Range: Text: Test reference range text High: 20 Test comment</text>
```

Comment:
Should this mention "High" when the interpretation code is low (that's if this is related)? I'm not too familiar with the logic behind this